inference-engine/src/cldnn_engine/ops/non_max_suppression.cpp

```diff
@@ -74,6 +76,9 @@ void CreateNonMaxSuppressionIEInternalOp(Program& p, const std::shared_ptr<ngrap
                 DefaultFormatForDims(op->get_output_shape(2).size()),
                 CldnnTensorFromIEDims(op->get_output_shape(2)));
 
+            GPU_DEBUG_IF(debug_config->verbose >= 2) {
+                GPU_DEBUG_COUT << "[" << layer_type_name_ID(op) << ": mutable data]" << std::endl;
+            }
```

Comment:
Why did you ignore some other ops with mutable data/internal memory allocations, such as proposal, topk, greedy decoder?

Reply:
I just missed those ops. updated.